spec/acceptance/log_from_organizer_spec.rb

```diff
@@ -151,4 +153,38 @@ def collects_log
       expect(log_message).not_to include(organizer_log_message)
     end
   end
+
+  context "when overriding the global LightService organizer" do
+    let(:global_logger_organizer) do
+      Class.new do
+        extend LightService::Organizer
+        def self.call(context)
+          with(context).reduce([
+            TestDoubles::AddsOneAction,
+            TestDoubles::AddsOneAction,
+          ])
+        end
+      end
+    end
+
+    let(:custom_logger_organizer) do
+      Class.new do
+        extend LightService::Organizer
+        log_with Logger.new(LOG_DIR.join("overridden.log"))
+
+        def self.call(coffee, this_hot = :very_hot)
+          with(:milk => this_hot, :coffee => coffee)
+            .reduce(TestDoubles::MakesLatteAction,
+                    TestDoubles::AddsTwoActionWithFetch)
+        end
+      end
+    end
+    it "logs in own logger" do
+      global_logger_organizer.call
+      custom_logger_organizer.call(coffee: "Cappucino")
+
+      # Test that overriden.log was written into
+      binding.pry
+    end
+  end
```

Comment:
I like that you added tests. Can you please move it into a new spec file? Something specific to this feature.